phpunit.xml.dist

```diff
@@ -9,7 +9,7 @@
          convertNoticesToExceptions="true"
          convertWarningsToExceptions="true"
          stopOnFailure="false"
-         executionOrder="random"
```

Comment:
What's the problem with random execution?

Reply:
disable random execution because sometimes it gives problems with these packages with those rare errors.